examples/tutorial/02_Plotting.ipynb

```diff
@@ -563,7 +563,105 @@
    "source": [
     "Here it is clear that the most commonly detected events are light, and typically shallow.\n",
     "\n",
-    "# Exploring further\n",
+    "## Output Matplotlib or Plotly plots\n",
+    "\n",
+    "While the default plotting backend/extension of hvPlot is Bokeh, it also supports rendering plots with either [Plotly](https://plotly.com/python/) or [Matplotlib](https://matplotlib.org/). As Bokeh, Plotly is an interactive library that provides similar data exploring actions (pan, hover, zoom, etc.). You can decide to use Plotly instead of Bokeh if you prefer its look and feel, or if your organization's policy is to use that plotting library. Matplotlib is mostly a static plotting library, it is great at creating static images (e.g. PNG) that you can then easily save and share with your friends or coworkers, and embed in documents for publication.\n",
+    "\n",
+    "To load a plotting backend you can use the `hvplot.extension` function. The first backend you declare in the call to the function will be set as the default one."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "hvplot.extension('plotly', 'matplotlib')\n",
+    "\n",
+    "small_df.hvplot.scatter(x='longitude', y='latitude')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Once a backend is loaded with `hvplot.extension` you should use the `hvplot.output` function to switch from a backend to another."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "hvplot.output(backend='matplotlib')\n",
+    "\n",
+    "plot = small_df.hvplot.scatter(x='longitude', y='latitude')\n",
+    "plot"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Save and further customize plots\n",
+    "\n",
+    "You can easily save a plot with the `hvplot.save` function, which is particularly useful when you are using Matplotlib as your plotting backend."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "hvplot.save(plot, 'plot.png')"
```

Comment:
Can we also show SVG? I think that will address people who think these tools aren't for publication.

Reply:
I've added an example with a SVG output.